.env.example

```
@@ -54,3 +54,8 @@ export PROMO_LIST={}
 export NEXT_PUBLIC_VAPID_PUBLIC_KEY=
 export VAPID_PRIVATE_KEY=
 export NEXT_PUBLIC_VAPID_SUBJECT="mailto:hello@peanut.to"
+
+# one signal 
+export NEXT_PUBLIC_ONESIGNAL_APP_ID=
+export NEXT_PUBLIC_SAFARI_WEB_ID=
+export NEXT_PUBLIC_ONEIGNAL_WEBHOOK=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_ONEIGNAL_WEBHOOK key should go before the NEXT_PUBLIC_ONESIGNAL_APP_ID key

(UnorderedKey)